GeneratorInterface/Pythia8Interface/plugins/Pythia8Hadronizer.cc

```diff
@@ -19,6 +19,7 @@ using namespace Pythia8;
 
 #include "GeneratorInterface/Pythia8Interface/plugins/ReweightUserHooks.h"
 #include "GeneratorInterface/Pythia8Interface/interface/CustomHook.h"
+#include "GeneratorInterface/Pythia8Interface/plugins/TopRecoilHook.h"
```

Comment:
in https://github.com/cms-sw/cmssw/pull/42180, it's using "#include "TopRecoilHook.h"